cmd/frontend/internal/cli/serve_cmd.go

```diff
@@ -293,7 +295,12 @@ func Main(enterpriseSetupHook func(db database.DB, c conftypes.UnifiedWatchable)
 		return err
 	}
 
-	internalAPI, err := makeInternalAPI(schema, db, enterprise, rateLimitWatcher)
+	codeIntelServices, err := codeintel.InitServices()
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	codeIntelServices, err := codeintel.GetServices(codeintel.Databases{
		DB:          db,
		CodeIntelDB: codeIntelDB,
	})
	if err != nil {
		return nil, err
	}
```

After changing the import I think this should work. You may have to re-order this (or rename the db variable) to hit the right codeintel db.

Reply:
Note: this is what we do [here](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@361f3947878e40debece029ae8dab8af68292f64/-/blob/enterprise/cmd/frontend/internal/codeintel/services.go). I'll try to standardize these calls so that we only initialize them once (assuming that you don't have any runtime error from this as-is).

Reply:
I had to get a codeIntelDB so I copy&pasted `mustInitializeCodeIntelDB`. Sourcegraph starts fine with the changes.

Reply:
Tested locally to confirm - seems good! I think we might be connecting to the codeintel-db twice now when running enterprise. It works but might add a second connection pool to the same database (though unused since this API isn't hit yet).

I'm refactoring some of the http transport layer stuff in a bit, I'll submit a PR to make sure we only connect once from the OSS code. Right now initialization is in the enterprise handlers, which is why you had to create a new connection in the first place.